Rover/sailboat.h

```diff
@@ -116,4 +135,18 @@ class Sailboat
     uint32_t tack_clear_ms;         // system time when tack was cleared
     bool tack_assist;               // true if we should use some throttle to assist tack
     UseMotor motor_state;           // current state of motor output
+    // in irons variables ---
+    static constexpr int rotate_clockwise = -1;
+    static constexpr int rotate_anticlockwise = 1;
+    // the percentage of full rudder to apply in irons
+    // too much can stall the rudder and delay getting out of irons
+    static constexpr float in_irons_rudder_percent = 2.0f/3.0f;
+    // system time when in_irons started
+    uint32_t in_irons_start_ms = 0U;
+    // in irons state flag. Dont set directly
+    bool is_in_irons           = false;
+    // value between -1 and 1 to send to steering when in irons
+    float in_irons_rudder = 0.0f;
+     // target heading in radians when in irons
+    float in_irons_heading_rad;
```

Comment:
Maybe a `in_iorns` struct to keep these in, or a little sub class.

Reply:
done